extension/fts/test/test_files/fts_small.test

```diff
@@ -204,3 +204,20 @@ Catalog exception: QUERY_FTS_INDEX function does not exist.
 ---- 2
 0|0.271133
 3|0.209476
+
```

Comment:
Can you also add a test case on table name conflict (create a new table with the same name as internal table created by the fts index) ?

Reply:
I think we can't solve the conflict between user tables and internal tables right now. This should be solved using schema.